packages-exp/auth-exp/src/core/errors.ts

```diff
@@ -433,3 +431,112 @@ export const _DEFAULT_AUTH_ERROR_FACTORY = new ErrorFactory<
   AuthErrorCode,
   AuthErrorParams
 >('auth', 'Firebase', _prodErrorMap());
+
+/**
+ * A map of potential Auth error codes, for easier comparison with errors
+ * thrown by the SDK.
+ *
+ * @remarks
+ * Please note that you can't tree-shake individual keys
```

Comment:
The style guide is not fond of "please."  Just "Note" is helpful enough I think.

Reply:
@sam-gc Do you mind fixing it, so we can get it released in the next release?

Reply:
```suggestion
 * Note that you can't tree-shake individual keys
```